### PR TITLE
docs: add hacktoberfest blog post

### PR DESCRIPTION
**Pull request by pswu11**
This PR adds a blog post to share QuestDB's participation in Hacktoberfest

### CODE REVIEW

File: blog/2021-10-01-hacktoberfest-questdb.md [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+  - timeseries
+  - hacktoberfest
+  - java
+image: /img/blog/2021-10-01/banner-hacktoberfest-1.png
+tags: [hacktoberfest, github, swag, opensource]
+---
+
+
+
+Hacktoberfest 2021 is starting today! For the first time, QuestDB is participating as an open source project. We&#39;re super excited to meet with other open source contributors and maintainers.
```
Comment (bsmth) [inline blog/2021-10-01-hacktoberfest-questdb.md:21]: The HTML entities should be literals, so `&#39;` should be `'`

File: blog/2021-10-01-hacktoberfest-questdb.md [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+1. Go to open source repositories that opt in for Hacktoberfest:
+    - **QuestDB Core Project**: [https://github.com/questdb/questdb](https://github.com/questdb/questdb)
+    - **QuestDB Documentation**: [https://github.com/questdb/questdb.io](https://github.com/questdb/questdb.io)
+    - Or, look for other open source projects labeled with `hacktoberfest` in their topics
+1. If you&#39;re new to the project, look for open issues labeled with `good first issues` or `help wanted` to get started 
+1. Before you commit, don&#39;t forget to read `CONTRIBUTING.md` and follow the contribution guideline 👍 
+
+
+
+## 🎁 Tees or trees + QuestDB SWAG
```
Comment (bsmth) [inline blog/2021-10-01-hacktoberfest-questdb.md:38]: `Tees or trees + QuestDB SWAG` could we rewrite to:

`Tees, trees and QuestDB swag`

File: blog/2021-10-01-hacktoberfest-questdb.md [hunk 3]
```diff
@@ -0,0 +1,63 @@
...
+
+Once you reach the [contribution target](https://hacktoberfest.digitalocean.com/resources/participation) of **4 valid pull requests**, you can claim the reward from the official organizer! In addition, if you successfully contribute to QuestDB projects, we offer extra SWAG for you through our [SWAG program](https://questdb.io/community/)!
+
+To make sure that your pull request is valid, please follow Hacktoberfest&#39;s [quality standard](https://hacktoberfest.digitalocean.com/resources/qualitystandards).
+
+
+
+## ℹ️ Get support &amp; updates
```
Comment (bsmth) [inline blog/2021-10-01-hacktoberfest-questdb.md:44]: same here with the HTML entity, could we use `and` instead

**Review by bsmth — CHANGES_REQUESTED** [review]
Could you run `prettier` on this please?

File: blog/2021-10-01-hacktoberfest-questdb.md [hunk 4]
```diff
@@ -0,0 +1,91 @@
...
+  source contributors and maintainers. To celebrate this, we put together some
+  hints for you to get started.
+keywords:
+  - database
+  - opensource
+  - timeseries
+  - hacktoberfest
+  - java
+image: /img/blog/2021-10-01/banner-hacktoberfest-1.png
+tags: [hacktoberfest, github, swag, opensource]
```
Comment (bsmth) [inline blog/2021-10-01-hacktoberfest-questdb.md:18]: could we add 'community' and 'digitalocean' tags?

File: blog/2021-10-01-hacktoberfest-questdb.md [hunk 5]
```diff
@@ -0,0 +1,91 @@
...
+  Hacktoberfest 2021 is starting! We are super excited to meet with other open
+  source contributors and maintainers. To celebrate this, we put together some
+  hints for you to get started.
+keywords:
+  - database
+  - opensource
+  - timeseries
+  - hacktoberfest
+  - java
+image: /img/blog/2021-10-01/banner-hacktoberfest-1.png
```
Comment (bsmth) [inline blog/2021-10-01-hacktoberfest-questdb.md:17]: I would just use one image for both the banner and the post itself, feel free to choose which one

File: blog/2021-10-01-hacktoberfest-questdb.md [hunk 6]
```diff
@@ -0,0 +1,91 @@
...
+offer extra SWAG for you through our
+[SWAG program](https://questdb.io/community/)!
+
+To make sure that your pull request is valid, please follow Hacktoberfest's
+[quality standard](https://hacktoberfest.digitalocean.com/resources/qualitystandards).
+
+## ℹ️ Get support and updates
+
+Some questions might appear when you're trying to contribute to QuestDB
+projects, here are the places to get supports from our team:
```
Comment (bsmth) [inline blog/2021-10-01-hacktoberfest-questdb.md:79]: change to:

```
projects; here are the places to get support and hints from our team:
```

**Review by bsmth — DISMISSED** [review]
Nice, thanks 👍🏻

**Comment by pswu11** [comment]
@bsmth fixed the image, pls review, thanks!

**Pull request merged**